test/e2e-helm/e2e_helm_suite_test.go

```diff
@@ -45,8 +47,16 @@ var _ = BeforeSuite(func() {
 	tc, err = testutils.NewTestContext(testutils.BinaryName, "GO111MODULE=on")
 	Expect(err).NotTo(HaveOccurred())
 
-	By("creating a new directory")
-	Expect(tc.Prepare()).To(Succeed())
+	tc.Domain = "example.com"
+	tc.Group = "cache"
+	tc.Version = "v1alpha1"
+	tc.Kind = "Memcached"
+	tc.Resources = "memcacheds"
+	tc.ProjectName = "memcached-operator"
+	tc.Kubectl.Namespace = fmt.Sprintf("%s-system", tc.ProjectName)
+
+	By("copying sample to a temporary e2e directory")
+	Expect(exec.Command("cp", "-r", "../../testdata/helm/memcached-operator", tc.Dir).Run()).To(Succeed())
```

Comment:
```suggestion
	By("copying sample to a temporary e2e directory")
	Expect(exec.Command(“cp”, “-R”,
	“../../testdata/helm/memcached-operator/“, tc.Dir).Run()).To(Succeed())
```

Reply:
To cp the content of the sample to e2e-1234. 
The test will be executed in e2e-1234/

Reply:
The code in this PR creates `test/e2e-helm/e2e-agfv/{PROJECT,Makefile,helm-charts,...}`

If I apply your suggested changes, it creates `test/e2e-helm/e2e-agfv/memcached-operator` and the e2e fails.

There's no need to call `Prepare()`. Calling it before the `cp` command is what causes the nested `memcached-operator` to exist. The existing `cp` command effectively creates `e2e-agfv` and copies the contents of `./testdata/helm/memcached-operator/` into `e2e-agfv`.

Here's a simple reproducer:
```
#
# simulate ./testdata directory
#
$ mkdir from/
$ touch from/PROJECT

#
# don't pre-create dest; directly copy
#
$ cp -r from dest-no-nest
$ ls -l dest-no-nest
total 0
-rw-r--r-- 1 joe joe 0 Oct 29 09:29 PROJECT

#
# DO pre-create dest; copy after
#
$ mkdir -p dest-nest
$ cp -r from dest-nest
$ ls -l dest-nest
total 4
drwxr-xr-x 2 joe joe 4096 Oct 29 09:30 from
```

Reply:
Oh, it is true. I tested locally as well 👍 
Thank you for share.